modules/polygon/src/polygon-utils.d.ts

```diff
@@ -1,85 +1,120 @@
-import {TypedArray} from "@math.gl/core/";
+import {TypedArray} from '@math.gl/core/';
 
 export const WINDING: {
-    CLOCKWISE: number; 
-    COUNTER_CLOCKWISE: number
+  CLOCKWISE: number;
+  COUNTER_CLOCKWISE: number;
 };
 
+// Polygon representation where all positions are stored in one flat array.
+type FlatArray = TypedArray | number[];
```

Comment:
Maybe this type should go into core? `NumberArray`? 

A type that covers all flat numeric arrays? Flat seems to be the default for arrays, you could alias it here I suppose.

Reply:
Yes, Added NumberArray to core